client/src/components/Card/Card.js

```diff
@@ -1,13 +1,22 @@
-import React, { Component } from 'react';
+import React, {Component} from 'react';
```

Comment:
F.Y.I. `{ Component }`, spaces between functions/classes in import statement is more common.

Reply:
oh thanks.